opendistro-elasticsearch-sql.release-notes.md

```diff
@@ -1,37 +1,43 @@
+* Enhancement [#282](https://github.com/opendistro-for-elasticsearch/sql/pull/282): Support datetime functions: MONTH, DAYOFMONTH, DATE, MONTHNAME, TIMESTAMP, MAKETIME, NOW, CURDATE (issue: [#235](https://github.com/opendistro-for-elasticsearch/sql/issues/235))
+* Enhancement [#300](https://github.com/opendistro-for-elasticsearch/sql/pull/300): Support DISTINCT feature in SELECT clause (issue: [#294](https://github.com/opendistro-for-elasticsearch/sql/issues/294))
+
+#### Response
+* Enhancement [#334](https://github.com/opendistro-for-elasticsearch/sql/pull/334): Change the default response format to JDBC (issue: [#159](https://github.com/opendistro-for-elasticsearch/sql/issues/159))
+
+### Bugfixes
+* BugFix [#267](https://github.com/opendistro-for-elasticsearch/sql/pull/267): Fixed operatorReplace Integration Test (issue: [#266](https://github.com/opendistro-for-elasticsearch/sql/issues/266))
+* BugFix [#275](https://github.com/opendistro-for-elasticsearch/sql/pull/275): Fix issue that IP type cannot pass JDBC formatter (issue: [#272](https://github.com/opendistro-for-elasticsearch/sql/issues/272))
+* BugFix [#284](https://github.com/opendistro-for-elasticsearch/sql/pull/284): Fixed flaky test suite, that was breaking Github action build
+* BugFix [#295](https://github.com/opendistro-for-elasticsearch/sql/pull/295): Corrected the selected field names displayed in the schema of JDBC formatted response (issue: [#290](https://github.com/opendistro-for-elasticsearch/sql/issues/290))
+* BugFix [#296](https://github.com/opendistro-for-elasticsearch/sql/pull/296): Fixed functions work improperly with fieldvalue/constant param for current use (issues: [#279](https://github.com/opendistro-for-elasticsearch/sql/issues/279), [#291](https://github.com/opendistro-for-elasticsearch/sql/issues/291), [#224](https://github.com/opendistro-for-elasticsearch/sql/issues/224))
+* BugFix [#298](https://github.com/opendistro-for-elasticsearch/sql/pull/298): Fixed issue of log10 function gets inaccurate results （issue: [#297](https://github.com/opendistro-for-elasticsearch/sql/issues/297))
+* BugFix [#307](https://github.com/opendistro-for-elasticsearch/sql/pull/307): Fix the issue of column alias not working for GROUP BY (issue: [#299](https://github.com/opendistro-for-elasticsearch/sql/issues/299))
+* BugFix [#333](https://github.com/opendistro-for-elasticsearch/sql/pull/333): Fixed the issue of substring not working correctly when fieldname is put as <table>.<column> (issue: [#330](https://github.com/opendistro-for-elasticsearch/sql/issues/330))
+* BugFix [#337](https://github.com/opendistro-for-elasticsearch/sql/pull/337): Fix JDBC response for delete query (issue: [#131](https://github.com/opendistro-for-elasticsearch/sql/issues/131))
 
 ## 2019-10-29, Version 1.3.0
```

Comment:
Could you adjust the release notes of v1.3, cuz it looks not correct.

Reply:
Line 55:  add bacticks to `<table>.<column>`

Reply:
sure. escape with `<table>.<column>`